app/scripts/templates/confirm.mustache

```diff
@@ -30,6 +30,18 @@
       </div>
     {{/isOpenGmailButtonVisible}}
 
+    {{#isOpenHotmailButtonVisible}}
+      <div class="button-row">
+        <a href="{{{ hotmailLink }}}" class="button" target="_blank" id="open-hotmail" type="button">{{#t}}Open Hotmail{{/t}}</a>
+      </div>
+    {{/isOpenHotmailButtonVisible}}
+
+    {{#isOpenYahooButtonVisible}}
+      <div class="button-row">
+        <a href="{{{ yahooLink }}}" class="button" target="_blank" id="open-yahoo" type="button">{{#t}}Open Yahoo{{/t}}</a>
+      </div>
+    {{/isOpenYahooButtonVisible}}
```

Comment:
you can probably reuse a single block and just do, this will be a lot better in terms of actually using "templates":

```

 +    {{#isOpenWebmailButtonVisible}}
 +      <div class="button-row">
 +        <a href="{{{ webmailLink }}}" class="button" target="_blank" id="open-webmail" type="button">{{ webmailButton}}</a>
 +      </div>
 +    {{/isOpenWebmailButtonVisible}}
```